MEAnalysis/python/MEAnalysis_heppy.py

```diff
@@ -23,6 +23,9 @@
     print "Loading ME config from", os.environ["ME_CONF"]
     meconf = imp.load_source("meconf", os.environ["ME_CONF"])
     from meconf import Conf
+elif 0:
```

Comment:
OK, I will merge this now, but lets find a better way to configure crab with a different MEM config.